src/api/Comments.js

```diff
@@ -17,7 +18,7 @@ class Comments extends OffsetBasedAPI {
         if (!id) {
             throw new Error('Missing file id!');
         }
-        return `${this.getBaseApiUrl()}/files/${id}/comments`;
+        return `${this.getBaseApiUrl()}/files/${id}/comments?fields=${getCommentsFields()}`;
```

Comment:
You should be able to pass the fields in as a param to offsetGet instead of modifying this function

Reply:
Are you cool with making the param for fetchComments an object then? I only need to pass in fileID + fields but fields is the 3rd or 4th argument and the others have defaults.